specs/WinRT/WinRTAPIContracts.md

```diff
@@ -62,27 +74,49 @@ namespace Microsoft.Windows.System
     }
 }
 ```
-## 1.1 ship blocker
-Any APIs that changed between 1.0 and 1.1 need to add a contract or 1.1 will not ship.
 
-## Telemetry
-API specific telemetry will not be collected because it does not provide any additional information
-compared to the Windows App SDK version.
+# 4. Missing contracts is a ship blocker
+WinRT APIs without a contract are a shipblocking bug.
 
-## Gate check-ins
+## 4.1. Gate check-ins
 Because the contracts are used for API versions there will need to be work done in the build system
 to make sure
 1. All APIs have contracts
 2. Each contract is correct. This means version and name.
 3. All Contracts and dependencies work with each other.
-4. Contracts/Types in releases honor the source/binary/behavior compatibility rules e.g. 
+4. Contracts/Types in releases honor the source/binary/behavior compatibility rules e.g.
 contracts/types in a stable release doesn't change in future releases of the same major version
 (e.g. contracts/types in 1.1 don't change in 1.2, 1.3, ...)
 
 The builds that check for contracts compatibility are:
 1. Nightly
-2. Release 
+2. Release
 3. PR
 
-## Open issues
-None.
+# 5. Telemetry
+API contract version specific telemetry will not be collected because it does not provide any
+additional information compared to the Windows App SDK version.
+
+# 6. All Contracts in Windows App SDK
+The list of all contracts defined across Windows App SDK
+
+| Feature            | Repository    | Contract                        | Namespace                                            | Comment |
+|--------------------|---------------|---------------------------------|------------------------------------------------------|---------|
+| AccessControl      | windowsappsdk | AccessControlContract           | Microsoft.Windows.Security.AccessControl             |         |
+| AppLifecycle       | windowsappsdk | AppLifecycleContract            | Microsoft.Windows.AppLifecycle                       |         |
+| AppNotifications   | windowsappsdk | WindowsAppNotificationsContract | Microsoft.Windows.AppNotifications                   |         |
```

Comment:
I think @hulumane requested that the "Windows" prefix be removed from WindowsAppNotificationsContract as no other contract has it. Doing so would require that the contract name be change in the corresponding idl. Since the contract has only been released as part of preview1, now (before preview2 is probably the best time to do it,

Reply:
Yes please : )

Reply:
Unless there was a specific reason to use Windows* in the prefix, i don't see any other contracts using the same.

Reply:
> I think @hulumane requested that the "Windows" prefix be removed from WindowsAppNotificationsContract

Super! That was one I questioned. Done.

>Since the contract has only been released

Not a problem. We have until 1.1 Stable, though sooner is better (and once 1.1 ships we're pretty much locked until 2.0).

Don't worry about what's currently checked in. We have a wide mix - some types have the desired contract (per above), some have a different one (e.g. DynamicDependency lacks the 'Contract' suffix) and some (most) have none. Goal is to get this list through API Review, then we can update whatever code necessary.